tvb_storage/tvb/storage/storage_interface.py

```diff
@@ -34,15 +34,20 @@
 
 .. moduleauthor:: Robert Vincze <robert.vincze@codemart.ro>
 """
+
 import os
+import shutil
 import uuid
+from cgi import FieldStorage
 from datetime import datetime
+from cherrypy._cpreqbody import Part
```

Comment:
Could we avoid the cherrypy dependency at storage level?

Reply:
Yes, the method were cherrypy is used is called only once and we can keep that method outside of StorageInterface, for example in export_manager.